docs/preact-testing-library/setup.md

```diff
@@ -0,0 +1,228 @@
+---
+id: setup
+title: Setup
+sidebar_label: Setup
+---
+
+`preact-testing-library` does not require any configuration to be used (as
+demonstrated in the example above). However, there are some things you can do
+when configuring your testing framework to reduce some boilerplate. In these
+docs we'll demonstrate configuring Jest, but you should be able to do similar
+things with [any testing framework](#using-without-jest) (preact-testing-library
+does not require that you use Jest).
+
+## Global Config
+
+Adding options to your global test config can simplify the setup and teardown of
+tests in individual files.
+
+### Cleanup
+
+You can ensure [`cleanup`](./api#cleanup) is called after each test and import
+additional assertions by adding it to the setup configuration in Jest.
+
+In Jest 24 and up, add the
+[`setupFilesAfterEnv`](https://jestjs.io/docs/en/configuration.html#setupfilesafterenv-array)
+option to your Jest config:
+
+```javascript
+// jest.config.js
+module.exports = {
+  setupFilesAfterEnv: [
+    'preact-testing-library/cleanup-after-each',
```

Comment:
It doesn't look like this file exists in the library: https://github.com/antsmartian/preact-testing-library/blob/master/